war/src/main/webapp/help/system-config/homeDirectory.html

```diff
@@ -1,17 +1,22 @@
 <div>
-  Jenkins stores all the data files inside this directory in the file system.
-  You can change this by either:
-  <ol>
+  Jenkins stores all of its data in this directory on the file system.
```

Comment:
by default Jenkins stores all of its information here...

The builds are data and can be stored somewhere else :)